rj_vision_receiver/src/vision_receiver.cpp

```diff
@@ -11,9 +11,8 @@
 #include <rj_utils/logging_macros.hpp>
 #include <rj_vision_receiver/vision_receiver.hpp>
 
-constexpr auto kVisionReceiverParamModule = "vision_receiver";
```

Comment:
Ah, I see. I think this variable can stay here instead of moving to `rj_constants`.